OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I believe you are adding those evil Byte Order Marks again using Visual Studio. At least I found quite a few during https://github.com/OpenRA/OpenRA/pull/7249 and it seems `OpenRA.Game.csproj` was BOM free before.